### PR TITLE
Refactor dislike/flagged response handling, clean-up and organizing test scripts

### PR DESCRIPTION
**Pull request by gracebrazil28**
Changes in main.py:
- Separated flagged and dislike response function definitions
- Updated chat handling for like/dislike responses
- Modified button click to trigger 'flagged' reaction
- Remove Commented CSV Logger functions, we want to query DB and then download CSV file

Changes in test_dynamodb_connection.py:
- Moved testing script to a test_scripts folder for better directory organization
- Added comments on the script's purpose and pass/fail conditions

### CODE REVIEW

**Comment by gracebrazil28** [comment]
@cs1090218 Separated dislike and flagged reaction handling, it was part of the demo from today.
I also removed the commented out manual CSV handling in main.py just to clean up a little bit. 
In the test script, I moved it to a new directory /test_script and also added comments on pass/fail condition (we can use later for deployment testing)

File: main.py [hunk 1]
```diff
@@ -179,19 +168,21 @@ def log_commended_response(history):
         log_chat_history(username, session_id, question, response, 'liked')
         gr.Info("Saved successfully!")
```
Comment (cs1090218) [inline main.py:169]: Could you add duration parameter here and in all other gr.Info popups? Reason being the duration param defaults to 10 and 10s is too long in my opinion. Maybe explicitly set duration to 2.

File: main.py [hunk 2]
```diff
@@ -179,19 +168,21 @@ def log_commended_response(history):
...
-    #     gr.Info("Saved successfully!", duration=0.5)
-
-    def log_flagged_response(history):
+    def log_disliked_response(history):
         if len(history) == 0:
             return
         response = history[-1][1]
         question = history[-1][0]
         log_chat_history(username, session_id, question, response, 'disliked')
         gr.Info("Saved successfully!")
```
Comment (cs1090218) [inline main.py:177]: Same here and rest of the gr.Info

**Comment by cs1090218** [comment]
Just one comment about adding duration parameter to gr.Info (see above)

**Review by cs1090218 — APPROVED** [review]

**Comment by gracebrazil28** [comment]
Usage in main.py:
`gr.Info("Saved successfully!", duration=2)`
While testing, I see this issue when adding gr.Info duration:

  File "../GitHub/IVY-testing-ui/ivy-testing-ui/main.py", line 169, in log_commended_response
    gr.Info("Saved successfully!", duration=2)
TypeError: Info() got an unexpected keyword argument 'duration'

This is similar to the fix that Rochan added in PR https://github.com/DILab-Ivy/ivy-testing-ui/pull/1
This is due to the boto3 version, it doesn't support it.

**Comment by cs1090218** [comment]
> Usage in main.py: `gr.Info("Saved successfully!", duration=2)` While testing, I see this issue when adding gr.Info duration:
> 
> File "../GitHub/IVY-testing-ui/ivy-testing-ui/main.py", line 169, in log_commended_response gr.Info("Saved successfully!", duration=2) TypeError: Info() got an unexpected keyword argument 'duration'
> 
> This is similar to the fix that Rochan added in PR #1 This is due to the boto3 version, it doesn't support it.

Got it, we can let it be then.

**Review by Rochan-HM — APPROVED** [review]
Looks good!
Thank you!

**Pull request merged**